packages/auth-nextjs/src/shared.ts

```diff
@@ -232,8 +239,9 @@ export abstract class NextAuth extends NextAuthHelpers {
                 req,
               );
             }
-            this.setAuthCookie(tokenData.auth_token);
-            cookies().delete(this.options.pkceVerifierCookieName);
+            await this.setAuthCookie(tokenData.auth_token);
+            const cookieStore = await cookies();
+            cookieStore.delete(this.options.pkceVerifierCookieName);
```

Comment:
Not a blocker: Probably a good enough reason to promote this (and the auth cookie delete) to it's own function. Seemed superfluous when it was a one liner before.

Reply:
sgtm, i'll name it `deletePkceVerifierCookie()`

Reply:
done